### PR TITLE
Added readme, comments to default app; Fixed typo

### LINKED ISSUE #58: Add docs for default app

### PR DESCRIPTION
**Pull request by karpp**
Closes #58

### CODE REVIEW

File: default/main.py [hunk 1]
```diff
@@ -70,6 +70,7 @@ async def run_queries(x_appengine_cron: Optional[str] = Header(None)):
         for user in models.Userdata.query():
             async with aiohttp.ClientSession() as httpSession:
 
+                # run VT query
```
Comment (niedziol) [inline default/main.py:73]: I don't know any Google guide for that, but I'm a big fan of writing comments using sentences.
So in this case it should be:
```suggestion
                # Run VT query.
```

File: default/main.py [hunk 2]
```diff
@@ -82,7 +83,8 @@ async def run_queries(x_appengine_cron: Optional[str] = Header(None)):
                         logging.error(f'VT query failed with {vt_resp.status}: {vt_resp.text()}')
                         raise HTTPException(400, 'Bad request')
 
-                    async with httpSession.post('https://webhook-dot-virustotal-step-2020.ew.r.appspot.com/',
+                    # auth to webhook, get JWT token
```
Comment (niedziol) [inline default/main.py:86]: Maybe something like that?
```suggestion
                    # Get JWT token from webhook app.
```

File: default/main.py [hunk 3]
```diff
@@ -93,6 +95,7 @@ async def run_queries(x_appengine_cron: Optional[str] = Header(None)):
                             logging.error(f'Authentication on webhook failed with {auth_resp.status}: {auth_resp.text()}')
                             raise HTTPException(403, 'Authentication failed')
 
+                        # send data with JWT to webhook
```
Comment (niedziol) [inline default/main.py:98]:
```suggestion
                        # Send data with JWT to webhook.
```

**Review by niedziol — CHANGES_REQUESTED** [review]

File: default/main.py (hunk 2 above)
Comment (karpp) [reply default/main.py]: I think it's different, because I'm not requesting static token, I'm asking for a personal token after authorisation

**Review by niedziol — APPROVED** [review]
LGTM

File: default/README.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+VT API key, VTI query and webhook URL are submitted by user via web form and stored in Google Cloud Datastore.
+
+## Authentication
+
+Authorization to webhook is done using private API key (stored in Cloud Secret Manager). The app first authorizes to webhook, receives JWT token, and then adds this token to its data sending requests.
+
+## Deployment
+
+There are _app.yaml_ configuration file and _cron.yaml_ for configuring cron that are being used for deployment to Google App Engine. Deploy can be done using `gcloud app deploy` command.
```
Comment (mgmacias95) [inline default/README.md:15]: Put the deployment command in a separate paragrah using ``` instead of `

**Review by niedziol — APPROVED** [review]
LGTM

**Review by mgmacias95 — APPROVED** [review]

**Pull request merged**